common/src/main/java/org/apache/hertzbeat/common/entity/manager/JsonLongListAttributeConverter.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.hertzbeat.common.entity.manager;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import jakarta.persistence.AttributeConverter;
+import org.apache.hertzbeat.common.util.JsonUtil;
+import java.util.List;
+
+/**
+ * json str to id list
+ */
+public class JsonLongListAttributeConverter implements AttributeConverter<List<Long>, String> {
+    @Override
+    public String convertToDatabaseColumn(List<Long> attribute) {
+        return JsonUtil.toJson(attribute);
+
+    }
+
+    @Override
+    public List<Long> convertToEntityAttribute(String dbData) {
+        TypeReference<List<Long>> typeReference = new TypeReference<>() {};
+        return JsonUtil.fromJson(dbData, typeReference);
```

Comment:
here we should consider the old db data `long` to convert to `list<Long>` , refer https://github.com/dromara/hertzbeat/blob/master/common/src/main/java/org/apache/hertzbeat/common/entity/manager/JsonTagListAttributeConverter.java#L39